### ISSUE TITLE
Please update to work on V15 FW

### ISSUE THREAD

**Issue by bigdaddy51200** [blawar/mmlinx#10]
Please update to work on V15 FW

**Comment by blawar** [comment]
Can you test this attached build, thx,
[050C91DBD7AE8000.zip](https://github.com/blawar/mmlinx/files/9871422/050C91DBD7AE8000.zip)

**Comment by bigdaddy51200** [comment]
I will . I will report back to you in just a few.

**Comment by bigdaddy51200** [comment]
still a no go. it tries to load then goes back to home screen.

**Comment by blawar** [comment]
Are there any error messages or anything? Or just flashes back to home screen?

**Comment by bigdaddy51200** [comment]
No errors. I see nintendo logo in bottom right  , like normal load, then
goes back to home

**Comment by bigdaddy51200** [comment]
shoot me a message on facebook messenger and I will send you vid if you
want.
[image: image.png]

**Comment by blawar** [comment]
Please try this build:
[050C91DBD7AE8000-2.zip](https://github.com/blawar/mmlinx/files/9872119/050C91DBD7AE8000-2.zip)

**Comment by bigdaddy51200** [comment]
the app loaded. I installed the bcd files from this course but they don't
show up in maker 2 .

20221001 Super Mario Bros 5 MikeyMike World By Sshadow [01009B90006DC000]

**Comment by blawar** [comment]
Did you have the right user selected in mmlinx before you injected?

**Comment by bigdaddy51200** [comment]
I only have 1 profile on the switch. it shows it in lower left on mmlinx

**Comment by bmxhooligan** [comment]
Same for me. No levels inject. Both are on the same user profile

**Comment by popostef** [comment]
bonjour, de même, rien ne s'injecte dans mario

**Comment by MadMario81** [comment]
Hello, 
I´m using smm2 302 with 15.0 mmlinx. Level don´t inject correctly in smm2. When I download from mmlinx I get the message saved to slot "x" (x=the next free one slot"). And when I check in current levels in mmlinx, the level names are written in strange characters (some are chars are asian symbols and other symbols). Also if I save a level in smm2, linx seems to find  it in current levels (but with strange chars in name), and once I download a level with mmlinx and go back to smm2, the list with levels is empty. It looks like there is a corruption at somepoint with the level save "maybe".

**Comment by JimZiii** [comment]
Any progress getting this to work on FW 15 yet?
I'm having the same issues, downloaded levels are all titled in weird characters and when i load MM2 the level list is completely empty

**Comment by lemon5ky** [comment]
same problem. Would be great if you can fix it.

Also could you share the path for levels to upload it manually to sd card?

**Comment by dialtone187** [comment]
blawar plz update this awesome app for fw 15+ so the levels will inject correctly

**Comment by giuseppe1010** [comment]
Has this been fixed? Mine isn’t working

**Comment by ElioMcCartney** [comment]
maybe an update to atmosphere 16? dont work here this

**Comment by jeyjai** [comment]
> maybe an update to atmosphere 16? dont work here this

He never really updated it to work on 15, so don't expect it to get an update for 16

**Comment by ElioMcCartney** [comment]
> > maybe an update to atmosphere 16? dont work here this
> 
> He never really updated it to work on 15, so don't expect it to get an update for 16

Work with Checkpoint... only put the file levels at checkpoint/saves

MM2 at version base

**Comment by MYusufY** [comment]
Yes the same problem here. It doesn't show up in mario maker unfortunately. Will it be fixed?

**Comment by lemon5ky** [comment]
any updates or forks with this plugin?

**Comment by ElioMcCartney** [comment]
Have saves an stores at tinfoil with inject saves... Or by transfers saves to switch, this fork don't work

Obter o Outlook para Android<https://aka.ms/AAb9ysg>
________________________________
